views/trainer/show.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Trainer Name: <%=@trainer.name%> </h1>
+<p> Level: <%=@trainer.level%> </h1>
+<img src="<%=@trainer.img_url%>">
```

Comment:
include spacing with whatever is in your erb tags, the same way you have been with html. so <%= @trainer.level %>